tests/model_hub_tests/tensorflow/test_tf_convert_model.py

```diff
@@ -63,6 +110,34 @@ def load_model(self, model_name, model_link: str):
         concrete_func._backref_to_saved_model = load
         return concrete_func
 
+    def compare_results(self, fw_outputs, ov_outputs):
+        fw_eps = 5e-2
+
+        # TF FE loses output structure in case when original model has output dictionary, where values are tuples.
+        # OV generates in this case a list of tensors with inner tensor names.
+        # Comparison of results in this case can be performed if output signature is known.
+        # TODO: When OV supports output dictionary of tuples need to remove usage of output signature here and update comparison method - Ticket TODO
+        if len(fw_outputs) != len(ov_outputs):
+            assert self.output_signature is not None, "The number of outputs between framework and OpenVINO is different and could not obtain output signature."
+            return compare_by_signature(ov_outputs, fw_outputs, self.output_signature, fw_eps)
```

Comment:
let us do repacking of OV results to the correct format for such case instead. I mean to have output dict where it maps some name to a tuple (or list).
After that below, perform comparison where you should update it and check cur_ov_res is a list then compare list.

Reply:
Done.